src/rustdoc_cmd.rs

```diff
@@ -207,4 +207,15 @@ mod tests {
             )
             .expect("no errors");
     }
+
+    #[test]
+    fn rustdoc_for_crate_in_workspace() {
+        RustdocCommand::default()
+            .dump(
+                Path::new("./test_rustdoc/crate_in_workspace/crate1/Cargo.toml"),
+                None,
+                true,
+            )
+            .expect("no errors");
+    }
```

Comment:
@skyzh might it make sense to add a second test case, similar to this one but using the workspace's `Cargo.toml` and passing a package spec instead of that `None`? I believe that should work too, if all goes well.

Reply:
sure, I'll update the test case

Reply:
done